### PR TITLE
Dev env tweaks

### PR DESCRIPTION
**Pull request by jkp85**

### CODE REVIEW

File: appdj/settings/dev.py [hunk 1]
```diff
@@ -16,20 +16,32 @@
...
 STATIC_ROOT = os.path.join(BASE_DIR, "static")
 
+MIGRATION_MODULES.update({
+    'silk': 'appdj.migrations.silk',
+})
+
 
 RAVEN_CONFIG = {
-    'dsn': f'https://{os.getenv("SENTRY_DSN")}@sentry.io/241782',
+    'dsn': os.getenv("SENTRY_DSN"),
```
Comment (johngriebel) [inline appdj/settings/dev.py]: We need to make sure this gets updated in the corresponding configs in devops

Comment (jgwerner) [reply appdj/settings/dev.py]: @johngriebel as long as the `env` file has the update, upon merge an issue for devops / onpremise should get opened automatically.

**Comment by jgwerner** [comment]
@jkp85 don't we need to update the `env` file with new key(s)?

File: appdj/settings/dev.py (hunk 1 above)
Comment (johngriebel) [reply appdj/settings/dev.py]: Env is modified by this PR though.

Comment (johngriebel) [reply appdj/settings/dev.py]: @jgwerner no, it's changing how that particular key is defined. It already exists. That's why I made the comment I did.

Comment (jgwerner) [reply appdj/settings/dev.py]: @johngriebel got it, thanks for clarifying.

**Comment by jkp85** [comment]
@jgwerner This PR won't affect prod version in any way. Those changes will only affect dev environment. There are no new environment variables introduced. We can redefine `SENTRY_DSN` variable to reflect this change, but it's not necessary at this point.

**Comment by johngriebel** [comment]
@roachmd I think you may have commented on the wrong PR

**Pull request closed without merging**